src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -3205,8 +3205,10 @@ private static void NativeCompletionProcessCommands(string wordToComplete, strin
                     if (pattern.IsMatch(completionText))
                     {
                         var listItemText = completionText;
```

Comment:
I've always thought the user experience wasn't quite right with this completion.

I wonder if it would be better to use `listItemText` instead of `tooltip` to show the process name **and** the process id.